docs/i18n/overview.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: i18n Overview
+name: Overview
+parent: i18n
+nav_order: 2
+---
+
+# Overview i18n Overview
+Many companies use SAP systems and applications and many different countries.
+To ensure easy translation, the "i18n" (Internationalization) process is used.
+The app is not always recreated, but only the texts are translated and replaced.
+
+## Language Tags
+To identify the individual languages, so-called "Language Tags" are used which are defined by [BCP-47] (http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry).
+These language tags are stored in the SAP system in table T002 for all stored languages.
+These correspond to the ["ISO 639 alpha-2 language code"](https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes). The SAPUI5 application then converts the determined language key into a BCP-47 tag.
+
+> As a best practice, it should be defined that the BCP-47 language code
+> should be kept as short as possible. When creating it, you should
+> avoid regions, script or subtags as far as possible - unless you add
+> important and helpful information.
+> Good example:
+i18n.properties
+i18n_DE.properties
+Bad example:
+i18n.properties
```

Comment:
already in "good example"?

Reply:
i didn't know how best to format.
I wanted to use "good/bad" in order not to forget it.
Should I take it out?

Reply:
https://github.github.com/gfm/#tables-extension- ?

Reply:
?

![image](https://user-images.githubusercontent.com/13335743/104437958-b62cb680-558f-11eb-93d5-86a16fe8c3d0.png)